bindings/kepler.gl-jupyter/DEVELOPMENT.md

```diff
@@ -0,0 +1,205 @@
+```shell
+pip install jupyter
+pip install notebook==6.0.1
+```
+
+#### 3. Install GeoPandas
+
+- Using conda
+```shell
+conda install geopandas
+```
+
+- Using pip
+
+```shell
+pip install geopandas
+```
+
+### Download and run keplergl in your local Jupyter Notebook
+
+#### Clone Repo
+```shell
+git clone https://github.com/keplergl/kepler.gl.git
+```
+
+### Setup JS
+#### 1. Install Js dependencies
+```sh
+cd bindings/kepler.gl-jupyter
+cd js
+yarn
+```
+
+Rerun `yarn` if any js dependency has changed (usually after pulling new changes from master).
+
+#### 2. Load mapbox token
+Add a [Mapbox access token](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/) to the environment.
+
+```sh
+export MapboxAccessTokenJupyter=<your_mapbox_token>
+```
+
+#### 3. Build the js module
+
+For development of the js code, it is convenient to start a local server to watch for changes:
+
+```shell
+yarn start
+```
+
+If you edit a file in the `js` directory, the local server will pick up those changes.
+
+To build without a local server that watches for changes, use
+
+```shell
+yarn build
+```
+
+### Setup jupyter
+
+#### 1. Install python module and enable extension from local files
+This command must be run **AFTER** the `js` setup, and folder `./keplergl/static/` was created. It only needs to be run once.
+
+```sh
+# dev install from folder containing setup.py
+pip install -e .
+
+# only needed in dev mode, not in normal mode.
+jupyter nbextension install --py --symlink --sys-prefix keplergl
+
+# only needed in dev mode, not in normal mode.
+jupyter nbextension enable --py --sys-prefix keplergl
+```
+
+NOTE:
+The above command `jupyter nbextension install -py --symlink --sys-prefix keplergl` is trying to create a symoblic link from the folder `bindings/kepler.gl-jupyter/keplergl/static` into jupyter's folder `nbextensions`.
+Please check if there folder `nbextensions/kepler-jupyter` already exists and if so, remove (or rename) it first.
+
+To find the location of `nbextensions` folder, you can use the following command:
+```shell
+realpath $(dirname $(dirname $(which jupyter)))/share/jupyter/nbextensions
+```
+For example for
+```
+$ where jupyter
+/Users/test/opt/anaconda3/envs/test37/bin/jupyter
+```
+the nbextensions should be at:
+```
+/Users/test/opt/anaconda3/envs/test37/share/jupyter/nbextensions
+```
+
+
+#### 2. Start jupyter notebook
+
+```shell
+cd notebooks
+jupyter notebook
+```
+
+
+#### Have fun!
+
+You can now start editing the `.js` and `.py` files to see changes reflected in your local notebook. After changing files in the `./js` folder, the local start script will recompile the `.js` files and put them in to `keplergl/static` folder. You need to reload the jupyter notebook page to reload the files.
+
+
+#### 3. Development for JupyterLab
+
+To target JupyterLab instead of Jupyter Notebook, use
+
+```shell
+pip install -e .  # in this folder
+cd js && jupyter labextension develop . --overwrite
```

Comment:
this fails for me as well -- `build` in `js` works but `develop` in this directory fails.